mrscake/src/App.js

```diff
@@ -10,10 +12,8 @@ import ContactUs from './components/ContactUs';
 import ProductTable from './components/ProductPage';
 import NavBar from './components/NavBar';
 import FooterPage from './components/Footer';
-import './App.css';
```

Comment:
why do you erase app.css?

Reply:
Agree, this is strange, moreover you add new styles to this file

Reply:
i added thats why i deleted

Reply:
I added two styles in app.cs and i will add app.cs in next commit, I was checking something without app.cs.

Reply:
I added app.cs.. see below